src/compiler/cpp_generator.cc

```diff
@@ -1164,8 +1164,8 @@ void PrintHeaderServerMethodStreamedUnary(
         *vars,
         "// disable regular version of this method\n"
         "::grpc::Status $Method$("
-        "::grpc::ServerContext* context, const $Request$* request, "
-        "$Response$* response) override {\n"
+        "::grpc::ServerContext* /* context */, const $Request$* /* request */, "
```

Comment:
Thanks for the changes. 

Please make whitespace consistent across all of these. Some of the changes look like "/* context */" and some look like "/*context*/". I prefer the latter.